modules/etcd-creating-automated-backups.adoc

```diff
@@ -0,0 +1,320 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc
+
+:_content-type: PROCEDURE
+[id="creating-automated-etcd-backups_{context}"]
+= Creating automated etcd backups
+
+The automated backup feature for etcd supports both recurring and single backups. Recurring backups create a cronjob that starts a single backup each time it triggers.
+
+You can set the method of backup retention to `RetentionNumber`, `RetentionSize`, or `""`. If unspecified or set to `""`, the default retention type is `RetentionSize` with 15 maximum backups. Specifying a retention type overrides the default value.
+
+Use dynamically-provisioned storage to keep the created etcd backup data in a safe, external location if possible. If dynamically-provisioned storage is not available, consider storing the backup data on an NFS share to make backup recovery more accessible.
+
+:FeatureName: Automating etcd backups
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a `FeatureGate` custom resource (CR) file named `etcd-enable-automated-backups.yaml` with the following contents:
++
+[WARNING]
+====
+The `CustomNoUpgrade` feature set required for this procedure is **not supported**, **cannot be undone**, and **prevents upgrades**.
+====
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    release.openshift.io/create-only: "true"
+  name: cluster
+spec:
+  customNoUpgrade:
+    enabled:
+      - AutomatedEtcdBackup
+  featureSet: CustomNoUpgrade
+----
+
+. Run the following command to apply the CR and enable automated backups:
++
+[source,terminal]
+----
+$ oc apply -f etcd-enable-automated-backups.yaml
+----
+
+. (Optional) If needed, complete the following steps to start a single backup:
```

Comment:
not sure if that's what @dusk125 mentioned, but this seems kinda out of place. Can we make another top level section for one-off backups?

In the future it should replace the section that's at the top of the page: "Backing up etcd data". Not sure if it makes sense to mention it in there too.

Reply:
I think it makes the most sense to put this into a separate procedure. Having all of these instructions in a single procedure is confusing IMHO.

Reply:
For now, I'm adding section headings to help make this easier to navigate. We can circle back around later as the feature matures and take a look at the docs again as needed.